crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -125,6 +125,14 @@ where
         // anyone pushing onto this, because the handles will be unset
         &mut self.children
     }
+
+    pub fn len(&self) -> usize {
+        let mut result: usize = 0;
+        for child in self.children() {
```

Comment:
This is possibly better as `self.children().iter().map(|child| child.len()).sum()`